Src/AutoNSubstituteUnitTest/SubstituteAttributeTest.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Linq;
+using Xunit;
+using Xunit.Extensions;
+
+namespace Ploeh.AutoFixture.AutoNSubstitute.UnitTest
+{
+    public class SubstituteAttributeTest

```

Comment:
The key logic tested here is the applicability of the attribute to the same code elements (properties, parameters and fields) that are also supported by the `SubstituteAttributeRelay`. Unfortunately the test name `AttributeCanBeAppliedToParametersOfTestMethods` is not very accurate after collapsing several facts into a single theory. Assuming it is corrected, if you still feel that its not valuable enough, I can remove the class. Would you like me to do that?